app/gui/src/ide/integration.rs

```diff
@@ -71,6 +72,53 @@ impl Model {
             });
         }
     }
+
+    /// Open a project through project manager API.
+    pub fn open_project(&self, project_name: &str) {
+        let logger = self.logger.clone_ref();
+        let controller = self.controller.clone_ref();
+        let name = project_name.to_owned();
+        crate::executor::global::spawn(async move {
+            if let Ok(managing_api) = controller.manage_projects() {
```

Comment:
The function does nothing if we are opened in cloud (where we cannot manage projects). Please add at least warning in that case (in else block of this if)

Reply:
Added a warning to the log